bundles/AdminBundle/Controller/Admin/MiscController.php

```diff
@@ -768,4 +768,29 @@ public function testAction(Request $request)
     {
         return new Response('done');
     }
+
+    /**
+     * @Route("/translate", name="pimcore_admin_misc_translate")
+     *
+     * @param Request $request
+     *
+     * @return JsonResponse
+     */
+    public function translate(Request $request)
+    {
+        $values = explode(',', $request->get('values'));
```

Comment:
Could be an issue if the value contains a `,` - I'd prefer a more generic approach, e.g. using `values[]=value` or escaping (urlencoding) the values. WDYT?

Reply:
I will use urlencoding